src/Compilers/CSharp/Test/Syntax/Parsing/RoleParsingTests.cs

```diff
@@ -3128,26 +3180,220 @@ public void WithMember_Const_WithoutImplicitOrExplicit()
                     }
                 }
                 N(SyntaxKind.OpenBraceToken);
-                N(SyntaxKind.FieldDeclaration);
+                N(SyntaxKind.CloseBraceToken);
+            }
+            N(SyntaxKind.EndOfFileToken);
+        }
+        EOF();
+    }
+
+    [Fact]
+    public void ExplicitInterfaceOperators()
```

Comment:
Let's have a version with `implicit`/`explicit` conversion operators, as that should appear nearly identical to an extension type until we see the `.`.

Reply:
I'm not sure what you mean. For conversion operators, the `operator` keyword must come right after `implicit`/`explicit`. The first opportunity for an `extension` token is after `operator`, so it wouldn't look much like an extension.
Added tests to illustrate.